frontend/src/components/fields/MoleculeSelector/MoleculeSelector.tsx

```diff
@@ -54,7 +59,13 @@ export const MoleculeSelector: React.FC<MoleculeSelectorProps> = ({
           {...params}
           variant="standard"
           fullWidth
-          label={isGeisa ? "GEISA 2020 Molecule" : "HITRAN 2020 Molecule"}
+          label={
```

Comment:
Just use Molecule everywhere. No selector. Code will be easier, more minimal result